### PR TITLE
reduce log spam by logging the failed detection only once

### PR DESCRIPTION
**Pull request by aryszka**
Signed-off-by: Arpad Ryszka <arpad.ryszka@gmail.com>

### CODE REVIEW

**Comment by aryszka** [comment]
:+1:

File: dataclients/kubernetes/clusterclient.go [hunk 1]
```diff
@@ -37,6 +37,8 @@ const (
 	serviceAccountRootCAKey    = "ca.crt"
 )
 
+const routeGroupsNotInstalledMessage = "RouteGroups CRD is not installed in the cluster."
```
Comment (szuecs) [inline dataclients/kubernetes/clusterclient.go]: please point to user documentation https://opensource.zalando.com/skipper/kubernetes/routegroups/

Comment (aryszka) [reply dataclients/kubernetes/clusterclient.go]: ok

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**